bitnami/memcached/templates/hpa.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if and .Values.Values.autoscaling.enabled (eq .Values.architecture "high-availability") }}
```

Comment:
I think there is a typo here: `.Values.Values.autoscaling.enabled`

Reply:
Yeah, leftovers from find/replace.